### PR TITLE
Docs validation

### PR DESCRIPTION
**Pull request by zhijianli88**
#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind ci
> /kind cleanup
> /kind dependency-change
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -472,7 +472,10 @@ uninstall:
 	rm -f $(OCIUMOUNTINSTALLDIR)/crio-umount.conf
 	rm -f $(CRICTL_CONFIG_DIR)/crictl.yaml
 
-docs-validation:
+docs-validation: bin/crio bin/crio-status
+	(bin/crio-status md | diff docs/crio-status.8.md - && \
+	bin/crio -d "" --config="" md | diff docs/crio.8.md -) || \
+	(echo "Docs are not updated. Please run \`make docs-generate\` to update commands documentation"; false)
```
Comment (saschagrunert) [inline Makefile:478]:
```suggestion
	(echo "Docs are not updated. Please run \`make docs-generation\` to update commands documentation"; false)
```

**Review by zhijianli88 — COMMENTED** [review]
This patch is totally wrong. 
i will drop it.

**Comment by zhijianli88** [comment]
Since present CI have already included a validate-docs CI job to validate docs, not sure this PR is still needed.
Feel free to close this.

**Comment by saschagrunert** [comment]
> Since present CI have already included a validate-docs CI job to validate docs, not sure this PR is still needed.
> Feel free to close this.

I agree it would be a double-check. Anyways, thank you for your effort! :pray: 
/close

**Pull request closed without merging**